styles/reset.js

```diff
@@ -0,0 +1,59 @@
+import { createGlobalStyle } from 'styled-components';
```

Comment:
this file already exists in `@irvingjs/styled`. that might change in the future, but we could reference it there for now.